chef/cookbooks/barclamp/libraries/nic.rb

```diff
@@ -605,6 +605,15 @@ def miimon=(millisecs)
       self
     end
 
+    def xmit_hash_policy
+      sysfs("bonding/xmit_hash_policy")
```

Comment:
/sys/class/net/<foo>/bonding exists for *bonds*, not all NICs. modprobe bonding on your laptop will create a bond0 with no slaves, and that will have a bonding directory. Furthermore, that function is in the Nic::Bond class, not the generic Nic class.